go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	github.com/hashicorp/consul v1.4.1 // indirect
 	github.com/hashicorp/go-gcp-common v0.0.0-20180425173946-763e39302965 // indirect
 	github.com/hashicorp/go-hclog v0.0.0-20190109152822-4783caec6f2e // indirect
+	github.com/hashicorp/go-multierror v1.0.0
```

Comment:
Presumably this shouldn't be here anymore?

Reply:
I think this is outdated, just pushed a new commit a bit afterwards with the result of `go mod tidy`.

Reply:
Yeah I noticed after I had submitted the comments, sorry.